compass/landice/tests/mismipplus/spin_up/streams.spin_up

```diff
@@ -0,0 +1,25 @@
+<streams>
+
+<immutable_stream name="restart"
+                  type="input;output"
+                  filename_template="restart.$Y.nc"
+                  filename_interval="output_interval"
+                  output_interval="0250-00-00_00:00:00"
+                  clobber_mode="truncate"
+                  precision="double"
+                  input_interval="initial_only"/>
+
+<stream name="output"
+        type="output"
+        filename_template="output.$Y.nc"
+        filename_interval="20000-00-00_00:00:00"
+        output_interval="00100-00-00_00:00:00"
+        precision="double"
+        clobber_mode="overwrite">
+</stream>
+
+<stream name="globalStatsOutput"
```

Comment:
It would be helpful to add `<var name="albanyVelocityError"/>` to the globalStats stream, so we can make sure the velocity solver is converging.

Reply:
Thanks for the suggestion here! I added this in [`7ef36a1`](https://github.com/MPAS-Dev/compass/pull/737/commits/7ef36a1d01d58112c6d06979deb4d13d8346b261).